egs2/nit_song070/svs1/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# spectrogram-related arguments
+fs=24000
+fmax=7600
```

Comment:
```suggestion
fmin=80
fmax=7600
```

Reply:
Just curious, is fmin used in following context besides pitch extraction?

Reply:
The one used in pitch extraction is `f0min`. `fmin` is used in feats extraction.

Reply:
Done.